src/Scene/Scene.js

```diff
@@ -75,6 +75,9 @@ define('Scene/Scene', [
 
         this.gLDebug = gLDebug;
         this.gfxEngine = c3DEngine(this,positionCamera,viewerDiv, debugMode,gLDebug);
+        this.needsUpdate = true;
+        this.forceUpdate = true;
```

Comment:
What are these boolean doing? I hope we can get rid of one of them because they seem redundant...